CMakeLists.txt

```diff
@@ -156,6 +156,115 @@ use_protobuf()
 
 setup_build_boilerplate()
 
+include(ExternalProject)
```

Comment:
Turns out this EP doesn't work if you happen to set DESTDIR when to run `make`, so I've made a few fixes in https://github.com/mellery451/rippled/commit/f861f7cdda2065c83baf217235afeb000f83b6ff.  That commit also stores the EP in a dir separate from the binary dir, so it's slightly more likely to be reused and not rebuilt each time. Please let me know if you have any issues with it.